src/main/resources/application.yaml

```diff
@@ -21,7 +21,7 @@ spring:
     allow-bean-definition-overriding: true
   datasource:
     driver-class-name: org.postgresql.Driver
-    url: ${SPRING_DATASOURCE_URL:jdbc:postgresql://localhost:5432/workallocation?gssencmode=disable}
+    url: ${SPRING_DATASOURCE_URL:jdbc:postgresql://localhost:5432/workallocation?sslmode=disable&gssencmode=disable}
```

Comment:
It needs to be outside the }, otherwise, it's just setting for the local env!

Or the property needs to be copied into a place where SPRING_DATASOURCE_URL is configured.